### PR TITLE
FAQ - update 'risk_info' entry for #971

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Change english heading to use "I get informed" instead of "you get information"
- "pseudonymized keys" -> "random IDs"
- Change "receive"/"empfangen" in 2. to "download"/"laden": Emphasize that the app is in control and not the server
- Rewrite the first 3 steps
- Introduce 4.: App assesses risk
- Point out that the solution architecture document is english-only

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -606,8 +606,8 @@
...
...th gesendeten Zufalls-IDs in das Corona-Warn-App Backend hochladen. Alle Geräte mit der Corona-Warn-App laden in regelmäßigen Abständen die Zufalls-IDs von diesem Backend herunter und prüfen lokal, ob sie einen übereinstimmenden Schlüssel via Bluetooth empfangen haben. Für das Hochladen und Herunterladen der Zufalls-IDs muss Ihr Gerät Zugang zum Internet haben, zum Beispiel über WLAN oder mobile Daten.</li><li>Gibt es eine Übereinstimmung zwischen einem vom Server heruntergeladenen und einem lokal gespeicherten Schlüssel, dann bedeutet das, dass Sie eine Begegnung mit einer Person hatten, die positiv auf Corona getestet wurde. Für diesen Abgleich von Übereinstimmungen benötigen Sie keine Bluetooth-Verbindung.</li><li>Auf Basis der gefundenen Risikobegegnungen errechnet die App Ihren persönlichen Risikostatus, welcher bestimmt, ob SIe ein \"niedriges Risiko\" (grüne Karte) oder \"hohes Risiko\" (rote Karte) angezeigt bekommen. Weitere Informationen zur Risikoberechnung finden sie im FAQ-Eintrag <a href='encounter_criteria' target='_blank' rel='noopener noreferrer'>Nach welchen Kriterien werden Begegnungen bewertet und wie unterscheiden sich die Handlungsempfehlungen?</a></li></ol>",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:609]:
```suggestion
                            "<ol><li>Alle Geräte, die die App installiert haben, senden und empfangen Zufalls-IDs, die dann lokal auf dem Gerät gespeichert werden. Die Bluetooth-Verbindung ist nur für das Senden und Empfangen der Zufalls-IDs notwendig.</li><li>Falls Sie positiv getestet wurden, können Sie Ihre über Bluetooth gesendeten Zufalls-IDs in das Corona-Warn-App Backend hochladen. Alle Geräte mit der Corona-Warn-App laden in regelmäßigen Abständen die Zufalls-IDs von diesem Backend herunter und prüfen lokal, ob sie einen übereinstimmenden Schlüssel via Bluetooth empfangen haben. Für das Hochladen und Herunterladen der Zufalls-IDs muss Ihr Gerät Zugang zum Internet haben, zum Beispiel über WLAN oder mobile Daten.</li><li>Gibt es eine Übereinstimmung zwischen einem vom Server heruntergeladenen und einem lokal gespeicherten Schlüssel, dann bedeutet das, dass Sie eine Begegnung mit einer Person hatten, die positiv auf Corona getestet wurde. Für diesen Abgleich von Übereinstimmungen benötigen Sie keine Bluetooth-Verbindung.</li><li>Auf Basis der gefundenen Risikobegegnungen errechnet die App Ihren persönlichen Risikostatus, welcher bestimmt, ob Sie ein \"niedriges Risiko\" (grüne Karte) oder \"hohes Risiko\" (rote Karte) angezeigt bekommen. Weitere Informationen zur Risikoberechnung finden sie im FAQ-Eintrag <a href='encounter_criteria' target='_blank' rel='noopener noreferrer'>Nach welchen Kriterien werden Begegnungen bewertet und wie unterscheiden sich die Handlungsempfehlungen?</a></li></ol>",

```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

File: src/data/faq.json [hunk 2]
```diff
@@ -600,12 +600,12 @@
...
...ur device. A Bluetooth connection is only necessary for sending and receiving random IDs.</li><li>If you are diagnosed with COVID-19, you can upload the random IDs you sent via Bluetooth to the Corona-Warn-App backend. At regular intervals, all devices with the app download these random IDs from the backend and check locally if they have received a matching key via Bluetooth. For uploading and downloading the random IDs, your phone must be able to access the Internet, for example, using WiFi or mobile data.</li><li>If there is a match between a key downloaded from the server and a key saved locally on your device, this means you were exposed to a person who has been diagnosed with COVID-19. Matching the encounters does not require a Bluetooth connection.</li><li>Based on the matched encounters, the app calculates your personal risk score that determines whether you get displayed a \"low risk\" (green) or \"high risk\" (red) card in the app. You can find out more about the risk calculation in the FAQ entry <a href='encounter_criteria' target='_blank' rel='noopener noreferrer'>By which criteria are encounters evaluated and how do the recommendations for action differ?</a></li></ol>",
```
Comment (Ein-Tim) [inline src/data/faq.json:608]: iOS requires active Bluetooth to carry out risk checks.

Comment (MikeMcC399) [inline src/data/faq.json:608]: "If you are diagnosed with COVID-19, you can upload the random IDs **you sent** via Bluetooth to the Corona-Warn-App backend."

suggest:

"If you are diagnosed with COVID-19, you can upload the random IDs your device sent via Bluetooth to the Corona-Warn-App backend."

Comment (MikeMcC399) [inline src/data/faq.json:608]: The link to bookmark
a href='encounter_criteria' 
does not work. It should be
a href='#encounter_criteria'

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [inline src/data/faq_de.json:609]: The link to bookmark
a href='encounter_criteria' 
does not work. It should be
a href='#encounter_criteria'

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
The links to the FAQ article en/faq/#encounter_criteria and de/faq/#encounter_criteria need to be corrected.

I also made a suggestion for changed wording.

File: src/data/faq.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq.json]: added a "between two devices" for the first sentence and removed the second one.

Comment (heinezen) [reply src/data/faq.json]: fixed

Comment (heinezen) [reply src/data/faq.json]: fixed

File: src/data/faq_de.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq_de.json]: fixed

Comment (heinezen) [reply src/data/faq_de.json]: fixed

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@heinezen @MikeMcC399 @Ein-Tim Thanks.

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Pull request merged**